src/sonic-host-services/scripts/hostcfgd

```diff
@@ -987,16 +994,32 @@ class HostConfigDaemon:
         # Initialize AAACfg
         self.hostname_cache=""
         self.aaacfg = AaaCfg()
+
+        # Initialize cache
+        self.cache = {}
         
 
     def load(self):
+        features = self.config_db.get_table('FEATURE')
+        self.feature_handler.update_all_features_config(features)
+
         aaa = self.config_db.get_table('AAA')
         tacacs_global = self.config_db.get_table('TACPLUS')
         tacacs_server = self.config_db.get_table('TACPLUS_SERVER')
         radius_global = self.config_db.get_table('RADIUS')
         radius_server = self.config_db.get_table('RADIUS_SERVER')
+        self.cache = {'AAA': aaa, 'TACPLUS': tacacs_global, 'TACPLUS_SERVER': tacacs_server,
```

Comment:
Should we have a cache for feature , interfaces, ntp related tables?

Reply:
Yes, why to avoid those?